core/server/services/bulk-email/index.js

```diff
@@ -168,7 +168,7 @@ module.exports = {
             // update batch success status
             return await emailBatchModel.save({
                 status: 'submitted',
-                provider_id: sendResponse.id
+                provider_id: sendResponse.id.trim().replace(/^<|>$/g)
```

Comment:
This `replace` is buggy producing `undefined` on both sides as a result, example from debug session:
```
 sendResponse.id.trim().replace(/^<|>$/g)
'undefined20210219001342.1.2107B24C283F9A12@sandbox1a102ce1e8514fb597612c97d40b08ac.mailgun.orgundefined'
```
Creates faulty records in the db

Reply:
I guess it's just missing an empty string as the second argument or the replace call

Reply:
🤦🏻‍♂️ thanks, fixed!